themes/bootstrap3/templates/Auth/ILS/loginfields.phtml

```diff
@@ -0,0 +1,16 @@
+<?php $loginMethod = $this->auth()->getManager()->getLoginMethod(); ?>
```

Comment:
The `getLoginMethod()` seems to be missing.

Reply:
Is this a typo for getILSLoginMethod?

Reply:
Yes, indeed. I failed to test with ILS and only used MultiILS. Now fixed, but I'm having some issues preventing me from actually testing it.

Reply:
Thanks! Hopefully @betullam can try again now.

Reply:
@demiankatz and @EreMaijala - Yes that was it, now it's working.